AmplifyPlugins/Core/AWSPluginsCore/Model/GraphQLDocument/GraphQLMutation.swift

```diff
@@ -29,7 +29,8 @@ public struct GraphQLMutation: SingleDirectiveGraphQLDocument {
     }
 
     public init(modelSchema: ModelSchema) {
-        self.selectionSet = SelectionSet(fields: modelSchema.graphQLFields)
+        let fields = modelSchema.graphQLFields.filter { !$0.isReadOnly }
+        self.selectionSet = SelectionSet(fields: fields)
```

Comment:
Why wouldn't we return the readonly fields in the selection set? If I understand this correctly, in addition to not serving use cases that might actually need to see the readonly field, this means I would receive a mutation response that is incomplete--what if my readonly field is required?

Reply:
@palpatim, you are correct - the selection set should not be affected by this change.  That is, readonly fields should still be requested in the selection set.